src/TSTransformer/nodes/jsx/transformJsxAttributes.ts

```diff
@@ -67,96 +45,23 @@ function createJsxAttributeLoop(
 	return statement;
```

Comment:
GitHub won't let me leave this comment on the proper line because it's not modified, but the `undefined` check should actually use `createTruthinessChecks` - TS allows spreading any _falsy_ types, not just undefined.

Reply:
Also, should this include a diagnostic check for `noMacroObjectSpread`?

Reply:
Actually, I'll move both of these fixes into #2546